### ISSUE TITLE
PackedTransaction Unpack() failed if action data contains eos.Symbol

### ISSUE THREAD

**Issue by fufesou** [eoscanada/eos-go#129]
file: decoder.go
func: func (d *Decoder) Decode(v interface{}) (err error) {}

The switch case block below is missing, witch causes a decode error.
```
func (d *Decoder) Decode(v interface{}) (err error) {
    ...
    case *Symbol:
                   var symbol *Symbol
                   symbol, err = d.ReadSymbol()
                   rv.Set(reflect.ValueOf(*symbol))
                   return
    ...
}
```

**Comment by firesWu** [comment]
same issue

**Comment by maoueh** [comment]
Should be fixed now that #131 has been merged, thanks @firesWu!
